src/lxterminal.c

```diff
@@ -451,24 +451,23 @@ static void terminal_close_tab_activate_event(GtkAction * action, LXTerminal * t
  * Close the current window. */
 static void terminal_close_window_activate_event(GtkAction * action, LXTerminal * terminal)
 {
-    int close_flag = 0;
-    if (terminal->terms->len > 1)
+    int close_flag = 1;
+    if (!get_setting()->disable_confirm)
```

Comment:
Consider using `&&` instead of nested `if`. The code is overnested and the changeset looks too much.